packages/fxa-settings/src/components/Settings/PageSettings/en.ftl

```diff
@@ -1,2 +1,4 @@
 # Link to delete account on main Settings page
 delete-account-link = Delete account
+# Success message displayed in alert bar after the user has successfully confirmed their account is not inactive.
+inactive-update-status-success-alert = Signed in successfully. Your Mozilla account and data will stay active.
```

Comment:
This should be: `Signed in successfully. Your { -product-mozilla-account } and data will stay active."